hack/install_workspaces.sh

```diff
@@ -4,9 +4,10 @@ set -e
 
 export QUAY_NAMESPACE=${QUAY_NAMESPACE:-workspaces}
 
-f=$(mktemp --directory /tmp/workspaces-demo.XXXX)
+f="$(pwd)"
+[[ "${f}" == "/tmp/*" ]] || {
+  f=$(mktemp --directory /tmp/workspaces-demo.XXXX)
+  cp -r hack/ operator/ e2e/ server/ ingress/ "$f" 
+}
```

Comment:
that's to avoid changing files in the current folder and having to revert the change before committing all the time. Probably not the best solution for this annoying scenario 🙃